Castle.Transactions.sln

```diff
@@ -65,38 +65,38 @@ Global
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.MONO28-Debug|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.MONO28-Release|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.MONO28-Release|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.NET35-Debug|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.NET35-Debug|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.NET35-Release|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.NET35-Release|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.NET40-Debug|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.NET40-Debug|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.NET40-Release|Any CPU.ActiveCfg = NET40-Release|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.NET40-Release|Any CPU.Build.0 = NET40-Release|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.Release|Any CPU.ActiveCfg = NET40-Release|Any CPU
-		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.Release|Any CPU.Build.0 = NET40-Release|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.Debug|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.Debug|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.MONO28-Debug|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.MONO28-Debug|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.MONO28-Release|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.MONO28-Release|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.NET35-Debug|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.NET35-Debug|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.NET35-Release|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.NET35-Release|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.NET40-Debug|Any CPU.ActiveCfg = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.NET40-Debug|Any CPU.Build.0 = NET40-Debug|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.NET40-Release|Any CPU.ActiveCfg = NET40-Release|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.NET40-Release|Any CPU.Build.0 = NET40-Release|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.Release|Any CPU.ActiveCfg = NET40-Release|Any CPU
-		{76DF5D09-FC04-411F-B123-01A6B9432796}.Release|Any CPU.Build.0 = NET40-Release|Any CPU
+		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{C19A20DD-8F32-44EF-9B68-43C70677F92D}.MONO28-Debug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
You can remove all the other build configurations, since you upgraded to .NET 4.6.1.